src/Kliva/Views/MainPage.xaml

```diff
@@ -122,11 +122,11 @@
             <Button x:Name="FilterButton"
                     Style="{StaticResource EmptyButtonStyle}"
                     Foreground="{StaticResource KlivaMainBrush}"
-                    Content="{Binding FilterText}"
-                    FontFamily="{StaticResource OpenSansFontLight}"
                     HorizontalAlignment="Stretch"
                     HorizontalContentAlignment="Left"
-                    Margin="10,5,0,10"
+                    Margin="10,0,0,0" Height="48"
+                    VerticalContentAlignment="Bottom"
+                    Padding="0 0 0 5"
```

Comment:
Pieter,
Het toevoegen van een drop down arrow vind ik perfect!
Maar kan je de alignment van de filter terug zetten zoals ervoor? Want indien je in mobile mode komt en er is geen detail zichtbaar, dan is er nu teveel wit ruimte bovenaan!

Denk dat we het beter ander proberen op te lossen en de titel van detail aan te passen?

Wat denk je...